### ISSUE TITLE
Lab5 Using tensorboard to track your training process

### ISSUE THREAD

**Issue by shwgao** [shwgao/CS-ECE-599-labs#6]

**Comment by pavlohav** [comment]
This was actually really cool to see your training loss and images that it was actively training as well as the progress of how it was able to categorize it.
<img width="738" alt="Image" src="https://github.com/user-attachments/assets/3b7cc9cb-c7b4-45fa-8b4d-97180e5bccdd" />

I meant a larger learning rate instead of lower

<img width="1014" alt="Image" src="https://github.com/user-attachments/assets/29bce9eb-db88-4859-8f14-fd54725ca2e1" />

<img width="277" alt="Image" src="https://github.com/user-attachments/assets/b73420b6-39e2-4c6f-947f-669c6d8ed941" />

osuid: 7425

**Comment by megant2321** [comment]
After going through this lab, the additional features for tensorboard are very interesting. Being able to see the structure of the model specifically is very interesting and I am hoping to add that feature to our term project. It would be helpful in visualizing the structure and previous to this lab I was not aware we could do that. So in the future, I am hoping to add that to our profiler for our term project!

osuid:7072

**Comment by drcrockerjr** [comment]
We implemented this into our CS2 prediction training procedure. It is very helpful to be able to view the performance metrics from your training. Additionally, it is helpful since you can have your metrics and profiler results in one place. Then if you wanted to see how two training iterations differed from one another. You can add two different tensorboard output files into one directory and view the differences from your changes.

OSU ID: 3645

**Comment by NotJeremyCapps** [comment]
We use tensor board for our model to help us understand our models training process. This allows us to check the effectiveness of our training and make tweaks depending on the training process. This allows us to have more transparency in the training process and have more control over the models training.
ONID:1556

**Comment by Maxttxxtt0110** [comment]
It is so cool to see tensor board providing all these performance measurement features, which help us ensure the effectiveness of our training processes. I would like to explore more possibility with these features.

ONID:5702
